herddb-core/src/main/java/herddb/model/commands/CheckpointStatement.java

```diff
@@ -0,0 +1,42 @@
+/*
+ Licensed to Diennea S.r.l. under one
+ or more contributor license agreements. See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership. Diennea S.r.l. licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+
+ */
+
+package herddb.model.commands;
+
+import herddb.model.Statement;
+
+/**
+ * TableSpace consistency check statement
+ *
+ * @author lorenzobalzani
+ */
+public class CheckpointStatement extends Statement {
+
+    private final boolean notEnqueue;
```

Comment:
what about `nowait` ?
I mean "run the checkpoint, but not not wait if it is already in progress"

cc @aluccaroni ?

Reply:
@eolivelli yeah, nowait is definitively better. Please, also document this command on the wiki under https://github.com/diennea/herddb/wiki/SQL-Support